data/json/recipes/recipe_others.json

```diff
@@ -3669,6 +3669,20 @@
     "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 50 ], [ "oxy_torch", 10 ] ] ],
     "components": [ [ [ "scrap", 3 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "railroad_track_small",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "1 h",
+    "autolearn": true,
+    "using": [ [ "steel_standard", 5 ] ],
```

Comment:
This needs forging_standard and anvil, hammer, etc (or blacksmithing_standard once that PR lands), as you're drawing out the steel into a long plate, I don't think that's a thing you can do without a forge.

Reply:
Thanks, I overlooked that. My bad. I wanted to add blacksmithing_standard but it was not merged so i'll have to do that once it it is in.